.github/workflows/pull-request.yaml

```diff
@@ -282,7 +282,7 @@ jobs:
             arch: amd64
             wasm: disabled
           - os: darwin
-            run: macos-latest
+            run: macos-13
             exec: opa_darwin_amd64
             arch: amd64
```

Comment:
Should we test opa_darwin_arm64_static too?

Reply:
It would be good if we were consistent with the macos version. If we just used `macos-13 ` then we don't need to change anything else.

Reply:
To test opa_darwin_amd64, we have to use macos-13.
And to test test opa_darwin_arm64_static, we have to use macos-14.

Reply:
I added a test of opa_darwin_arm64_static.

https://github.com/open-policy-agent/opa/pull/6721/commits/f41eccdda4a79b28324d2d96e6c5e35da3f78017